src/emit.iced

```diff
@@ -628,7 +637,7 @@ exports.GoEmitter = class GoEmitter
     @output "},"
 
   emit_server_hook_method_type : ({name, details}) ->
-    @output "MethodType: rpc.Method#{if is_one_way(details) then 'Notify' else 'Call'},"
+    @output "MethodTypes: []rpc.MethodType{#{if is_one_way(details) then 'rpc.MethodNotify' else 'rpc.MethodCall, rpc.MethodCallCompressed'}},"
```

Comment:
i'm pretty sure this is entirely unused, should we just kill it?

Reply:
ok, if unused, yeah...